Pod/Classes/MenuView.swift

```diff
@@ -85,21 +85,21 @@ public class MenuView: UIScrollView {
     
     // MARK: - Lifecycle
     
-    internal init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
+    public init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
```

Comment:
It works as static header for me. I use it in my project.
menuItemCount needs to be specifically set for now. 



```
let pageMenuOption: PagingMenuOptions = {
     let options = PagingMenuOptions()
     options.menuItemCount = 3
     return options
}()
displayInfoView = MenuView(menuItemTitles: ["title 1", "title 2", "title 3"], options: options)
self.addSubview(displayInfoView!)
```